src/SingleSelect/Input.js

```diff
@@ -54,6 +55,12 @@ const Input = ({
                     line-height: 16px;
                 }
 
+                .root-input {
+                    max-height: ${inputMaxHeight};
+                    overflow-y: auto;
+                    flex: 1;
+                }
```

Comment:
Same as previous comment. Needs to be extracted to a separate styled-jsx tag.

Reply:
See https://github.com/dhis2/ui-core/pull/580#discussion_r346335238